css/index.css

```diff
@@ -104,14 +180,47 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
+    /* background: black; */
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+/* Box colors */
```

Comment:
Good job getting the box colors and overriding only what you need to change for each box.